app/controllers/worktimelog_controller.rb

```diff
@@ -64,7 +64,7 @@ def issue_summary
     where_query = "issue_id = #{params[:issue_id]} AND flag = 1"
 
     if (params[:started] && params[:finished])
-      where_query += "AND (started, finished) OVERLAPS ('#{params[:started]}'::DATE, '#{params[:finished]}'::DATE)"
+      where_query += " AND (started >= '#{params[:started]}' AND finished <= '#{params[:finished]}')"
```

Comment:
Params should be transformed to the DATE otherwise they will be recognised as a string.